src/explore-education-statistics-admin/src/pages/release/datablocks/components/chart/ChartDataSetsConfiguration.tsx

```diff
@@ -55,6 +56,19 @@ const ChartDataSetsConfiguration = ({
     [meta.locations],
   );
 
+  const hasMixedUnits = useMemo(() => {
+    const units: string[] = [];
+    dataSets.forEach(dataSet => {
+      const foundIndicator = meta.indicators.find(
+        indicator => indicator.value === dataSet.indicator,
+      );
+      if (foundIndicator) {
+        units.push(foundIndicator.unit);
+      }
+    });
+    return !units.every(unit => unit === units[0]);
```

Comment:
Are we sure at this point that we've got at least one unit available in our `units` array at this point?  If not, this'll get an index out of bounds error, so maybe worth doubling up with a length check?

Reply:
If `units` is empty the condition is not evaluated. The docs say  `Calling this method on an empty array will return true for any condition`. I tested this with an empty array and hasMixedUnits returns `false` as expected.

Reply:
Haha ahh yes of course!  Thanks :)